packaging/ubuntu-1604/build-image/inside/build-package

```diff
@@ -15,13 +15,11 @@ cd /tmp/build
 rm -rf build-dir
 mkdir build-dir
 cd build-dir
-cmake -DREALM_ENABLE_ENCRYPTION=1 \
-      -DREALM_NO_TESTS=1 \
-      -DCMAKE_INSTALL_PREFIX=/prod \
-      -DCMAKE_BUILD_TYPE=Release \
-      -GNinja ..
-ninja
-ninja install/strip
+cmake -D CMAKE_INSTALL_PREFIX=/prod \
+      -D CMAKE_BUILD_TYPE=Release \
+      -G Ninja ..
+ninja Core RealmConfig RealmImporter Realmd
+ninja install
```

Comment:
Was the `install/strip` target a mistake here? I notice that stripping a release build still reduces the binary size, though I'm not sure if this will negatively affect stack traces of our crash reports.

Reply:
Fixed